### PR TITLE
[SDK, React, CLI] Make generate work with useContract

### PR DESCRIPTION
**Pull request by adam-maj**

### CODE REVIEW

File: packages/cli/src/generate/command.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ export async function generate(options: GenerateOptions) {
   const packagePath = `${projectPath}/node_modules/@thirdweb-dev/generated-abis/dist`;
   if (!fs.existsSync(packagePath)) {
     throw new Error(
-      `Could not find files at ${projectPath}/node_modules/@thirdweb-dev/generated-abis`,
+      `Unable to cache ABIs. Please ensure that you're using @thirdweb-dev/sdk > 4.0.0`,
```
Comment (jnsdls) [inline packages/cli/src/generate/command.ts:85]: wait, we're shipping v4?

File: packages/react-core/src/evm/hooks/async/contracts.ts [hunk 2]
```diff
@@ -19,13 +19,14 @@ import {
   useQueryClient,
   UseQueryResult,
 } from "@tanstack/react-query";
-import type {
+import {
```
Comment (joaquim-verges) [inline packages/react-core/src/evm/hooks/async/contracts.ts:22]: we had `import type` here before

File: packages/cli/src/generate/command.ts (hunk 1 above)
Comment (joaquim-verges) [reply packages/cli/src/generate/command.ts:85]: nope nope nope

Comment (adam-maj) [reply packages/cli/src/generate/command.ts:85]: This version will change to whatever minimum version actually has the generated-abis package as a hard dep

Comment (joaquim-verges) [reply packages/cli/src/generate/command.ts:85]: just put 'latest' version for now?

**Review by joaquim-verges — APPROVED** [review]
ship it

File: packages/cli/src/generate/command.ts (hunk 1 above)
Comment (joaquim-verges) [inline packages/cli/src/generate/command.ts:85]:
```suggestion
      `Unable to cache ABIs. Please ensure that you're using the latest @thirdweb-dev/sdk package`,
```

**Pull request merged**